src/bytes.rs

```diff
@@ -191,7 +191,9 @@ impl_atomic_access!(i32, std::sync::atomic::AtomicI32);
     target_arch = "x86_64",
     target_arch = "aarch64",
     target_arch = "powerpc64",
-    target_arch = "s390x"
+    target_arch = "s390x",
+    target_arch = "riscv64",
+   target_arch = "riscv64"
```

Comment:
This line looks duplicated

Reply:
Sorry for this mistake. This has been fixed now.